packages/rocketchat-api/server/helpers/replaceUserIdKeyByUserObject.js

```diff
@@ -0,0 +1,14 @@
+RocketChat.API.helperMethods.set('replaceUserIdKeyByUserObject', function _addUserToObject({ object, userId }) {
+	const { username, name } = RocketChat.models.Users.findOneById(userId);
+	if (object.hasOwnProperty('userId')) {
+		delete object.userId;
```

Comment:
I'm not sure if removing the userId is a good idea, as it may break something that relies on it, but I tested the application and didn't find any problem.